### PR TITLE
Pradeep 

### PR DESCRIPTION
**Pull request by Pradeep-microsoft**
[Bug 10549305](https://msazure.visualstudio.com/One/_workitems/edit/10549305): [Supporting Platform -Analytics-Color Contrast]: In High Contrast Black Mode, controls present in 'Analytics' page are not properly visible.

[Bug 13396902](https://msazure.visualstudio.com/One/_workitems/edit/13396902): A11y_IoT Central_Create dashboard_Add a tile with telemetry metrics_HighContrast:When the screen is in high contrast theme "Aquatic" the icons beside temperature pressure moisture are not visible

- Created new SVG file for dark theme  i.e Zoom in & Zoom out button.
- Added Dark theme mode  eye icon, and ellipses button

### CODE REVIEW

File: src/UXClient/Components/AvailabilityChart/AvailabilityChart.scss [hunk 1]
```diff
@@ -238,10 +238,16 @@
 
         .tsi-zoomButtonIn{
             background: url(../../Icons/ZoomIn_Light_Theme.svg) no-repeat;
+            @media screen and (-ms-high-contrast: white-on-black) { 
+                background: url(../../Icons/Zoomin_Dark_Theme.svg) no-repeat !important;
```
Comment (chengch801214) [inline src/UXClient/Components/AvailabilityChart/AvailabilityChart.scss:242]: Is the !important flag here needed?

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts [hunk 2]
```diff
@@ -452,18 +452,25 @@ class DateTimePicker extends ChartComponent {
...
+
                 .text(d => d);
+                
+              
         }
     }
 
     private rangeIsValid(prospectiveFromMillis: number, prospectiveToMillis: number) {
         var accumulatedErrors = [];
+        var errorDiolge;
```
Comment (chengch801214) [inline src/UXClient/Components/DateTimePicker/DateTimePicker.ts:466]: Where is this used?

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts [hunk 3]
```diff
@@ -547,7 +554,8 @@ class DateTimePicker extends ChartComponent {
                 .classed("tsi-timeLabel", true)
                 .attr('id', inputLabelID)
                 .attr('for', inputID)
-                .attr('aria-label', `${startOrEnd === 'start' ? this.getString('Start time input') : this.getString('End time input')}`)
+                .attr('required',true)
+                .attr('aria-label', `${startOrEnd === 'start' ? this.getString('Start time Checking') : this.getString('End time input')}`)
```
Comment (chengch801214) [inline src/UXClient/Components/DateTimePicker/DateTimePicker.ts:558]: Why change it to 'Start time Checking'?

File: src/UXClient/Components/Slider/Slider.scss [hunk 4]
```diff
@@ -8,6 +8,9 @@
    
     .tsi-sliderTrack {
         stroke: $gray1;
+        @media screen and (-ms-high-contrast: white-on-black) { 
+            stroke: white !important;
```
Comment (chengch801214) [inline src/UXClient/Components/Slider/Slider.scss:12]: Is this !important flag needed here? Since for the non-high-contrast stroke it isn't needed?

**Review by chengch801214 — CHANGES_REQUESTED** [review]

File: src/UXClient/Components/DateTimePicker/DateTimePicker.ts (hunk 3 above)
Comment (Pradeep-microsoft) [reply src/UXClient/Components/DateTimePicker/DateTimePicker.ts:558]: This is not required. I'll remove this changes

**Review by chengch801214 — APPROVED** [review]

**Pull request merged**